### ISSUE TITLE
[Feature] Rename the Tagger

### ISSUE THREAD

**Issue by JackDawson94** [stashapp/stash#2786]
**Is your feature request related to a problem? Please describe.**
"Tagger" does not appropriately describe what this button does. It should be renamed to make it's function more explicit.
The icon it uses is also linked to "Tagging" which is not what the feature is used for, so the icon should be changed too.

**Describe the solution you'd like**
Rename to "StashBox Sync" or "Sync with StashBox" (other ideas welcome)
Change the icon to something more appropriate ideas could be fa-rotate or fa-cloud-arrow-down maybe ?

**Describe alternatives you've considered**
Leave it as it currently is:  unclear, but overall it works

**Comment by 7dJx1qP** [comment]
I agree that the Tagger could use a better name. However, I don't think StashBox should be included in the name, since the scene Tagger is not just limited to stashbox. It can be used with autotag and any scraper that supports it. The performer Tagger is stashbox only, so your suggestions could work for that.

My suggestion for a rename would probably involve the word "Identify" since the Tagger is related to the Identify task and the function of both is to identify scenes. The Tagger could be called "Interactive Identify" while the identify task could be renamed "Mass Identify".

As for the icon, I think the fa-cloud-arrow-down or some variant of it is a good idea since the Tagger is pulling data down from somewhere (except for if autotag is the source). I don't like fa-rotate since people might think that's a reload/refresh button.

**Comment by scruffynerf** [comment]
Yeah, we began discussing this in Discord, and I agree with removing "Tag" names entirely, especially Autotag, which needs to be hidden from users who don't understand what it does or why.  It's only useful in limited circumstances, and not for newbies who all think it's the tool to use from the name.

**Comment by JackDawson94** [comment]
`Interactive Identify` (or Identifier ?) makes sense 👍
This would work for both Scenes & Performers (and maybe in the future Studios, Movies, ...)

In that case fa-cloud-arrow-down seems like a pretty good icon

**Comment by echo6ix** [comment]
If people want to make it super noob friendly maybe `Scene Lookup` ala MusicBrainz Picard's `Lookup`, and the tag icon is important because it's basically a universal symbol for metadata tagging.

> The icon it uses is also linked to "Tagging" which is not what the feature is used for, so the icon should be changed too.

I disagree. Tagging is assigning metadata or keywords to content, such as automatically or manually populating MP3 ID3 tag fields with values (artist name, album name, date, etc). This is precisely what the Tagger does and it's usage is in line with other apps that use the term (Beets, Mp3tag, MusicBee, Media Monkey, Foobar2000, etc).

I don't understand this community's recoil to the term

**Comment by echo6ix** [comment]
It seems there is also some confusion about the word tag based on the Discord thread. There's conflation with (1) the verb tag (to assign metadata to content), (2) the Stash category keywords called Tags (http://localhost:9999/tags), (3) and the noun tags (which is a synonym for metadata).

I think a lot of confusion would be alleviated by changing (2) to the word categories (http://localhost:9999/categories).

**Comment by scruffynerf** [comment]
While I agree in principle... The real world (that is discussion with multiple people on Discord... Many multiple people...) The tag icon is meaningless to them.  They don't see it as relevant visually.  When I point it out, they are usually surprised we didn't make it more obvious.

You are nit picking the wording, and assuming people all use the word tag the way you do.

They don't.  And that's our problem.  The user experience needs to drive the words and icons, not the other way around.

People don't come to Stash solely to tag, they come to get organized, to identify, all of these words together are basically the same thing, but using words like Autotag don't work.  So let's try something new and see if it helps.

Not going to argue further.  The many opinions of user feedback 
in #help means more to me than the opinion of an experienced user who can't see the UI with fresh eyeballs the way those new users do.  And saying they should know better or RTFM, or that better instructions is all we need is missing the point.

**Comment by echo6ix** [comment]
There's no need to personalize the discussion by making assumptions and accusations about one's inability to see a concept your way as a means to delegitimize their feedback. We should keep it civil. It's fine if you don't want to argue further, but that is how competing views deliberate if concepts have well founded reasons or not based on sound premises. 

> [...] The real world (that is discussion with multiple people on Discord... Many multiple people...) [...]

The implication here is that I am out of touch and the feedback received in #help is indicative of some massive problem. Maybe it is, maybe it isn't. I don't know if this is accurate without some kind of quantification, such as another user feedback survey. This could just be [selection bias](https://en.wikipedia.org/wiki/Selection_bias) -- that is, obviously the sample of people who have issues with parts of Stash are going to be more vocal about it in #help than users don't have issues, but it's not necessarily indicative of the majority of new users, and it would be fallacious to draw a conclusion simply based on that.

Understanding selection bias is an important principle of UX design. There's a ton of articles about, don't take my word for it.

> People don't come to Stash solely to tag, they come to get organized, to identify, all of these words together are basically the same thing, but using words like Autotag don't work. So let's try something new and see if it helps.

But how do we know they don't work without at least quantifying feedback of all new users? Again, you could be right or it could be selection bias. It might just be a small vocal percentage within a larger set.

> You are nit picking the wording, and assuming people all use the word tag the way you do.

But the issue that keeps being raised _is_ about nit-picking a term, and some reasons given to justify it are factually incorrect such as *"Tagger has nothing to do with what the module is doing"* and *"I've never seen the word tag used like that before"*, etc. Stash's usage of term is not only [factually correct](https://en.wikipedia.org/wiki/Tag_(metadata)), it's also [not uncommon](https://en.wikipedia.org/wiki/Tag_editor) among apps and various help docs of major apps that use the word  tagging to describe editing of metadata, including but not limited to [Kodi](https://kodi.wiki/view/Video_file_tagging), and throughout the Plex community which I don't consider some obscure niche community.

If the reason for perusing a decision is based on an unsound premises I don't consider it nitpicking to point that out.

I also realize usage of the term tag/tagging/tagger might be confusing to some people when Stash also refers to category keywords as *Tags* (http://localhost:9999/tags). Some people might get the impression that tagging only has to do with the category *Tags*. Stash isn't the only app that does this, and I recognize this isn't necessarily a good justification for doing it. I've addressed a solution to that on Discord and will restate it again below.

> And saying they should know better or RTFM, or that better instructions is all we need is missing the point.

Kind of misrepresenting the views I've advocated on your Discord thread:
- Educating the user through a user-friendly help guide within Stash that corresponds to the UI with images. I'm a so-called advanced user and even I find the help guide disorientating and unhelpful for many things
- Educating the user through a basic interactive orientation tutorial on first-run wrt basic functionality. Gmail does this almost every time they do UI refresh or introduce new features upon sign in. This isn't missing the point. This is addressing the point through education on first-run.
- Using basic design principles to address the point through:
  - Consistency in design. Consistent use of terms to reduce cognitive load and to ensure functionality is the same throughout the app. This includes
    - Renaming *Tags* (http://localhost:9999/tags) to *Categories* (http://localhost:9999/categories) to avoid ambiguity with the verb tag/tagging/tagger and the noun tag as used to denote metadata.
    - In settings, grouping all the forms of auto-tagging subroutines like *Identify*, *AutoTag*, and *Filename Scene Parser*. They're all forms of auto-tagging; their differentiation is how they're accessing metadata to apply the automatic tagging.  A great example of this is *paste* functionality in right click context menus in apps like Excel. Excel doesn't use completely different names to denote the different types of pasting options, rather all options are grouped together for consistency under *Paste options*, and differentiated within that subset to denote their forms of pasting.
![image](https://user-images.githubusercontent.com/37937507/182287616-da859e7a-7bff-4b32-b3b9-9103c9cdc0a8.png)
    - Agreeing that `Tagger` could have a better name for the module and suggesting `Scene Lookup` ala MusicBrainz Picard. Along with consolidating the term `Scrape With...` and whatever term is decided for `Tagger` because both are essentially doing the same thing (using an online source to interactively tag a scene with metadata).
    - Better describing what AutoTag does on the Performer/Studio/Tag pages by grouping it in a consistent presentation as the above image
![image](https://user-images.githubusercontent.com/37937507/182287527-7a54420d-ba9e-49c9-8ee8-e33af99a849e.png)
  - The use of symbols to convey meaning throughout the app (see above images)

TLDR: I think you're unfairly giving me a hard shake just for providing scrutiny. I'm not *just* suggesting that some of the premises stated are incorrect. I'm skeptical that it's a huge issue without quantifiable evidence (why is that bad?). And I'm providing tangible solutions based on basic design principles to address the stated issue beyond just renaming a few functions. Maybe I'm wrong, but hashing out ideas like this will hopefully make the app better.

**Comment by scruffynerf** [comment]
If you review the thread on discord, you weren't the only voice saying the above.  It wasn't personal.  It was factual and reflected multiple views expressed, not just yours.  You disagree with the multiple feedback, including literally someone in help popping up with an "autotag didn't work for me with stashDB" comment in #help right as we were were discussing the "autotag" problem in #sustainability.  If you want to believe it's not a real problem, that's on you. (And I mean that in a non personal way.).  Quoting how everyone else uses these terms is beyond the point.  It isn't working now, for users here, daily.

**Comment by Maista6969** [comment]
Can be closed as this has its own RFC in #4336
